src/ui/components/results/paginationcomponent.js

```diff
@@ -239,18 +245,18 @@ export default class PaginationComponent extends Component {
         desktopBack: this._pinFirstAndLastPage && desktopBackLimit > 0,
```

Comment:
Do we need to check that it's greater than 1 here? If the backLimit is 1, wouldn't we not want ellipses?

Reply:
this is an interesting point, I suppose it would be a bit odd to see 1 ... 2 3 4 5 6 ... 10

I'll update this so that numbers have to actually be skipped for the ellipses to show up.